src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -54,6 +44,16 @@ function getLandoConfig() {
 	const isAllDebugSelected = process.env.DEBUG === '*';
 	const logLevelConsole = ( isAllDebugSelected || isLandoDebugSelected ) ? 'debug' : 'warn';
 
+	const vipDir = path.join( xdgBasedir.data || os.tmpdir(), 'vip' );
+	const landoDir = path.join( vipDir, 'lando' );
+	const fakeHomeDir = path.join( landoDir, 'home' );
+
+	try {
+		await fs.promises.mkdir( fakeHomeDir, { recursive: true } );
+	} catch ( err ) {
+		// Ignore
+	}
```

Comment:
We do this to be compatible with `@lando/core` - until they accept our patch, at least.

Reply:
What's a fakeHomeDir?

Reply:
Empty directory to be mounted as `/home`. `@lando/core` will create the `.ssh` directory inside it. If we pass an empty string as `home` (see below), Lando will create `.ssh` directory in the current directory, which is not what we want.

In brief, we are trying to solve here the issue we solved in https://github.com/Automattic/lando-cli/pull/15, except that we don't want to fork `@lando/core`.